api/scpca_portal/test/management/commands/test_load_data.py

```diff
@@ -54,8 +59,29 @@ def assertProjectData(self, project):
         self.assertIsNotNone(sample.tissue_location)
         self.assertIsNotNone(sample.treatment)
 
-    def assertProjectReadmeContains(self, text, project_zip):
-        self.assertIn(text, project_zip.read("README.md").decode("utf-8"))
+    def assertProjectReadmeContent(self, zip_file):
+        def get_masked_content(content):
+            """
+            Replace the project ID and date with placeholders for format testing.
+            """
+            content = re.sub(r"SCPCP\d{6}", "SCPCPXXXXXX", content)
+            content = re.sub(
+                r"Generated on: \d{4}-\d{2}-\d{2}", "Generated on: YYYY-MM-DD", content
+            )
+
+            return content
```

Comment:
Let's update this to take content, project_ids, and replace the new placeholders in the test_data readmes with the values passed in. So that out expected output matches what we are generating in the tests.